test/functional/apps/dashboard/dashboard_state.ts

```diff
@@ -290,14 +290,17 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
         it('resets a pie slice color to the original when removed', async function () {
           const currentUrl = await getUrlFromShare();
-          const newUrl = currentUrl.replace(`vis:(colors:('80,000':%23FFFFFF))`, '');
+          const newUrl = isNewChartsLibraryEnabled
+            ? currentUrl.replace(`'80000':%23FFFFFF`, '')
+            : currentUrl.replace(`vis:(colors:('80,000':%23FFFFFF))`, '');
           await browser.get(newUrl.toString(), false);
           await PageObjects.header.waitUntilLoadingHasFinished();
 
           await retry.try(async () => {
-            const pieSliceStyle = await pieChart.getPieSliceStyle(`80,000`);
-            // The default green color that was stored with the visualization before any dashboard overrides.
-            expect(pieSliceStyle.indexOf('rgb(87, 193, 123)')).to.be.greaterThan(0);
+            const pieSliceStyle = await pieChart.getPieSliceStyle('80,000');
+            const color = isNewChartsLibraryEnabled ? 'rgb(111, 135, 216)' : 'rgb(87, 193, 123)';
```

Comment:
I've actually changed this test here:
https://github.com/elastic/kibana/pull/97941

I noticed it was a bit flaky when run locally, so in that PR I cached the original color before any modifications, then compared with the original style instead of a hardcoded color code. 

https://github.com/elastic/kibana/blob/997c6b061127e06e928efc6a398178371f1cb7b7/test/functional/apps/dashboard/dashboard_state.ts#L276

That should work in both cases here. Do you think it would be worth modifying this test to work that way in this PR?

Reply:
Of course my dear Devon, addressed that here https://github.com/elastic/kibana/pull/83929/commits/e96a5a1dfe8b7964836ecb2184062657c2876699
Thanx a lot for your review 🙂